packages/vitest-environment-nuxt/src/runtime/mount.ts

```diff
@@ -84,8 +84,14 @@ export async function mountSuspended<T>(
                     const clonedComponent = {
                       ...component,
                       render: render
-                        ? (_ctx: any, ...args: any[]) =>
-                            render(renderContext, ...args)
+                        ? (_ctx: any, ...args: any[]) => {
+                          // add all _ctx properties to renderContext
+                          // the renderContext must remain intact, otherwise the emits don't work
+                          for (const key in _ctx) {
```

Comment:
This change now triggers the following warning in my tests:

```log
Avoid app logic that relies on enumerating keys on a component instance. The keys will be empty in production mode to avoid performance overhead. 
```